backend/src/Designer/Controllers/OptionsController.cs

```diff
@@ -25,14 +26,17 @@ namespace Altinn.Studio.Designer.Controllers;
 public class OptionsController : ControllerBase
 {
     private readonly IOptionsService _optionsService;
+    private readonly IOrgCodeListService _orgCodeListService;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="OptionsController"/> class.
     /// </summary>
     /// <param name="optionsService">The options service.</param>
-    public OptionsController(IOptionsService optionsService)
+    /// <param name="orgCodeListService">The CodeList service for organisation level.</param>
```

Comment:
Her tror jeg det passer bedre med [fellesnavn](https://www.korrekturavdelingen.no/egennavn-fellesnavn.htm) (små bokstaver). Hva tenker du?
```suggestion
    /// <param name="orgCodeListService">The code list service for organisation level.</param>
```